src/DOMAssert.php

```diff
@@ -84,6 +84,26 @@ public static function assertSelectRegExp(
         );
     }
 
+    public static function assertXPathEquals(
+        string $selector,
+        ?string $content,
+        bool|int $count,
```

Comment:
```suggestion
        array|bool|int $count,
```
And also add the correspondent `@param` docblock